watson/cli.py

```diff
@@ -374,6 +376,12 @@ def status(watson, project, tags, elapsed):
         ))
         return
 
+    if elapsedminutes:
+        click.echo(u"{}".format(
+            style('time',str(int((arrow.utcnow() - current['start']).seconds/60)))
```

Comment:
```suggestion
            style('time', str(int((arrow.utcnow() - current['start']).seconds / 60)))
```

We should use Black to format / check the whole project code.